test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -1635,6 +1636,201 @@ public void executeTryToViewExams_personMissingInAddressBookErrorMessage() throw
                 lastShownList);
     }
 
+    @Test
+    public void executeClearSuccessWithExams() throws Exception {
+        TestDataHelper helper = new TestDataHelper();
+        // TODO: refactor this elsewhere
+        logic.setStorage(saveFile);
+
+        // generates the 3 test people and execute the add command
+        Person testPerson1 = helper.generatePerson(1, true, 1, false, 2);
+        Person testPerson2 = helper.generatePerson(2, true, 1, false, 2);
+        Person testPerson3 = helper.generatePerson(3, true, 3, false, 1);
+        addressBook.addPerson(testPerson1);
+        addressBook.addPerson(testPerson2);
+        addressBook.addPerson(testPerson3);
+
```

Comment:
If ya using only 3 person, it will be cleaner to use the ThreePerson class under TestDataHelper

Reply:
Yes i need to improve on this alot. I'm just testing everything works first then refactor and improving all.